docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.9'
+
+services:
+  ntt:
+    depends_on:
+      postgres:
+        condition: service_healthy
+    build:
+      context: .
+    restart: on-failure
+    environment:
+      - NEW_BUILD=true
+      - POSTGRES_HOST=postgres
+    ports:
+      - "5001:5001"
```

Comment:
I would be nice to expose a volume mount here. It's something I haven't done yet with Docker compose though. Not sure if it has to be done here or in the Dockerfile, but the idea is to re-compile if there are changes in the local repo